packages/amplify-schema-validator/src/validators/fields-passed-are-in-parent-model.ts

```diff
@@ -0,0 +1,64 @@
+import {
+  DocumentNode,
+  Kind,
+  ListValueNode,
+  ObjectTypeDefinitionNode,
+  StringValueNode,
+} from 'graphql';
+import { InvalidDirectiveError } from '../exceptions/invalid-directive-error';
+import { getObjectWithName } from '../helpers/get-object-with-name';
+import { resolveFieldTypeName } from '../helpers/resolve-field-type-name';
+
+/**
+   * Validates that any of the fields passed in are in the parent model
+   *
+   * @param schema graphql schema
+   * @returns true
+   */
```

Comment:
Could you also flesh out the documentation here a bit more too?

In general I wouldn't use the term parent model, Could we refer to it here as the specific relationship type, or if it applies to all relationship types, maybe just call it `validateFieldsMatchInRelatedModel`

Reply:
Fixed